notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<blockquote>we would pick three winning candidates along the Pareto curve:</blockquote><p>Please define <code>Parato curve</code>.This maybe obvious to us, but we're hoping to target users that may have no clue what terms like "QPS, recall, and Pareto frontier" mean.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>